enzyme/build.boot

```diff
@@ -12,7 +12,7 @@
 
 (task-options!
  pom  {:project     'cljsjs/enzyme
-       :version     +lib-version+
+       :version     "3.0.0-1"
```

Comment:
A pattern I've seen used in other libraries is to define :

```
(def +lib-version+ "3.0.0")
(def +version+ (str +lib-version+ "-1"))
```
Then use `+version+`  here